team/bundles/org.eclipse.jsch.ui/src/org/eclipse/jsch/internal/ui/authenticator/WorkbenchUserAuthenticator.java

```diff
@@ -143,10 +144,11 @@ public String[] promptForKeyboradInteractive(
 				prompt!=null && prompt.length==1 && prompt[0].trim().equalsIgnoreCase("password:")){ //$NON-NLS-1$
 			location.setPassword(result[0][0]);
 			if(location.getPasswordStore()!=null){
-				if(allowCaching[0])
+				if(allowCaching[0]) {
```

Comment:
@akurtakov I was just taking a look at this PR and wondered if we should aim at formatting the lines affected by the cleanups (here: adding missing whitewpace9 to avoid that we will have another round of changes to the same lines of code if we applied formatting later on? But probably it's fine to just apply the cleanups now and keep formatting for later.

Reply:
I would say let's stick to what we have now. The bigger the PRs the harder to review so I tend to prefer a series of smaller ones.